web/test/pages/index-test.js

```diff
@@ -0,0 +1,10 @@
+import IndexPage from '../../src/pages/index'
+import React from 'react'
+import { render } from '@testing-library/react'
+
+describe('<IndexPage />', () => {
+  it('renders the text', () => {
+    const { getByText } = render(<IndexPage />)
+    expect(getByText('Welcome to the Columbus Tip Jars')).toBeInTheDocument()
```

Comment:
I think this is fine since it's explicit, but I will just say that `toBeInTheDocument()` is redundant since `getByText` will throw if it doesn't find the text in the document.

Reply:
Yea, I think this just makes it more clear what the test is testing and makes it look like an actual test instead of just a random line of code